panel/widgets/chat.py

```diff
@@ -1370,8 +1374,7 @@ def active(self) -> int:
         """
         if isinstance(self._input_layout, Tabs):
             return self._input_layout.active
-        else:
-            return -1
+        return -1
```

Comment:
This removes one line from the code base which makes it faster to read for users and for python.